src/Template/Loader/KalastaticLibraryLoader.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\kalastatic\Template\Loader;
+
+/**
+ * Loads templates from the filesystem.
+ *
+ * This loader adds Kalastatic specific paths as namespaces to the Twig
+ * filesystem loader so that templates can be referenced by namespace.
+ */
+class KalastaticLibraryLoader extends \Twig_Loader_Filesystem {
+
+  // Keep track of libraries that we attempt to register.
+  protected $libraries = [];
```

Comment:
Is the libraries part actually needed? Since the variable is declared in the child, I don't think it will have any bearing on the parent.